deisctl/units/deis-logspout.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=deis-logspout
-Requires=docker.socket etcd.service
-After=docker.socket etcd.service
+Wants=docker.socket etcd2.service
+After=docker.socket etcd2.service etcd.service
```

Comment:
Yes, because `After=` is not binding. If `etcd.service` is disabled then systemd won't wait for it before launching logspout. CoreOS uses the same approach to make their own services backward-compatible with v1 etcd:
```console
core@deis-01 ~ $ systemctl cat fleet   
# /usr/lib64/systemd/system/fleet.service
[Unit]
Description=fleet daemon

After=etcd.service
After=etcd2.service

```